smpplib/command.py

```diff
@@ -123,7 +124,7 @@ def generate_params(self):
         if hasattr(self, 'prep') and callable(self.prep):
             self.prep()
 
-        body = ''
+        body = b''
```

Comment:
I think. it is better to define some byte-string constants and use them latter, like
in consts.py:
```
EMPTY_STRING=six.b('')
NULL_STRING=six.b('\0')
```